mteb/evaluation/MTEB.py

```diff
@@ -170,7 +175,28 @@ def _display_tasks(self, task_list, name=None):
 
     def mteb_benchmarks(self):
         """Get all benchmarks available in the MTEB."""
-        for benchmark in self._tasks:
+        from mteb.overview import MTEBTasks
+
+        # get all the MTEB specific benchmarks:
+        mteb_tasks = [t for t in self._tasks]
+
+        mteb_tasks, remaining_tasks = (
+            [t for t in self._tasks if "MTEB" in t.name],
+            [t for t in self._tasks if "MTEB" not in t.name],
+        )
+
+        mteb_tasks = sorted(mteb_tasks, key=lambda obj: obj.name.lower())
+        remaining_tasks = sorted(remaining_tasks, key=lambda obj: obj.name.lower())
+
+        sorted_tasks = mteb_tasks + remaining_tasks
+
+        # task ordering within each benchmark should be alphabetical
+        for st in sorted_tasks:
+            st.tasks = MTEBTasks(
+                sorted(st.tasks, key=lambda obj: obj.metadata.name.lower())
+            )
+
+        for benchmark in sorted_tasks:
```

Comment:
I think the sorted_tasks variable name is a bit ambiguous? sorted_benchmarks seems better